pages/community/events/conference.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 'UX Conference'
```

Comment:
```suggestion
title: 'AsyncAPI Conference'
```

are we really going to feature any UX conferences any time soon? 🧐  I guess I think it seems more realistic to name an event we for sure will want to show, like the AsyncAPI conference